### ISSUE TITLE
CGCNNFeaturizer Example

### ISSUE THREAD

**Issue by sgbaird** [hackingmaterials/matminer_examples#61]
I've been struggling a lot with getting CGCNNFeaturizer to work. The google groups doesn't seem to have much discussion on this, nor does the CGCNN repo have much to say about it other than a clarification about python bindings. The matminer documentation is also a bit hard to follow, but I think I've understood the main components. Does someone have (or could someone put together) a basic .ipynb showing how to get features from one of the pretrained models (e.g. 'bulk-moduli)?

**Comment by sgbaird** [comment]
Seems that it may have been a version incompatibility. Will probably put in a pull request once I polish the .ipynb up a bit.

**Comment by ardunn** [comment]
The CGCNN featurizer is incredibly difficult to use. We may add a new module with things better thought out for using graphnet models. Stay tuned on the main repo for progress on that front!
